main.py

```diff
@@ -176,8 +176,9 @@ async def list(message: Message):
         message_to_send += f"""
 {index+1}. {author.nick or author.name} - `{filename}`"""
 
-    # Send the message
-    await message.channel.send(message_to_send)
+    # Send the message and pin it
+    list_message = await message.channel.send(message_to_send)
+    await list_message.pin()
```

Comment:
This should work! And while not something that'll likely occur in Busty's, could you handle the documented error codes? https://discordpy.readthedocs.io/en/latest/api.html?highlight=pin#discord.Message.pin

Someone may forgot to give the "manage messages" permission to the bot for instance.

Reply:
Sure, added handling for the pin error codes